software/ryot.yml

```
@@ -0,0 +1,13 @@
+name: "ryot"
+website_url: "https://github.com/ignisda/ryot"
+source_code_url: "https://github.com/ignisda/ryot"
+description: "Platform for tracking various facets of your life - media, fitness etc."
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+tags:
+  - Analytics
+  - Miscellaneous
+  - Time Tracking
```

```suggestion
tags:
  - Personal Dashboards
```


More fitting category